lib/utils.js

```diff
@@ -109,6 +109,16 @@ exports.findStationByName = function (stationName) {
  * @param  {String} string to by titleize'd
  * @return {String} the titleize'd string
  */
-exports.titleize = function (str) {
+function titleize (str) {
   return str.replace(/\w+(\W|$)/g, match => _.upperFirst(match));
+}
+
+module.exports = {
```

Comment:
This uses the new es6 object shorthand for when the key and value have the same name.